client/src/styles/history-chat-responsive.css

```diff
@@ -9,7 +9,7 @@
 .history-chat-wrapper {
   width: 40vw;
   border-radius: 10px;
-  height: 100vh !important;
+  /* height: 90vh !important; */

```

Comment:
delete code, don't comment out and merge

Reply:
still see the commented code. We generally don't want to keep commented-out code at merge time. If we no longer want this line of code, delete it

Reply:
I removed it, sorry about that